### ISSUE TITLE
Cannot catch exceptions when using within a custom Command

### ISSUE THREAD

**Issue by guyinpv** [Propaganistas/Laravel-Phone#182]
I've created a custom Laravel command that loops over a phone number field in the database so that I can format the numbers and save back to the DB in E164 format.

I've run into an issue where some phone numbers fail because users entered them wrong. 
I'm formatting the number with this:
`$contact->value = PhoneNumber::make($contact->value, ['US'])->formatE164();`

The problem is, I have that code in a try/catch block, but whenever there is a failure to format, it crashes the script instead of catching the exception.

Two exceptions I get are "Number does not match the provided country" which is ` libNumberParseException::INVALID_COUNTRY_CODE`
And also "The string supplied did not seem to be a phone number." which is a `NumberParseException()`

Neither of these can be caught by try/catch no matter what I try. I tried catching as `\Exception` and as `\Throwable`. I tried catching the specific exception itself, no luck.

I need to catch all the exceptions so I can log and print to screen and skip that iteration, etc.

I can't tell if this is a Laravel issue or the phone script. I also can't call it a bug, as maybe it was done on purpose?

Bottom line, when using the formatter in a Laravel Command, can I catch all the possible exceptions? And if so, how?

This has been multiple days trying to figure out, including getting help from Laravel professions, it doesn't make sense to anybody why try/catch isn't working. It must be a "bubble up" issue of some kind.

**Comment by Propaganistas** [comment]
Here's a proof of concept showcasing that catching the exception in a vanilla Laravel installation works just fine.

https://github.com/Propaganistas/POC-Laravel-Phone-182

And here's the command class catching all exceptions: https://github.com/Propaganistas/POC-Laravel-Phone-182/blob/d474bec41cee96c6a2e47e931c10e06edf256096/app/Console/Commands/ProofOfConceptCommand.php#L41

I hate to say, but I don't think this issue is related to this package...

**Comment by guyinpv** [comment]
Thanks, that's very good to know.

Then it must have something to do with the other libraries or configuration. 

With some more testing, I was able to use try/catch from inside a normal controller, just not from the custom command controller.

Here are other libraries from composer:

```
"require": {
        "php": "^7.2",
        "aws/aws-sdk-php": "^3.158",
        "doctrine/dbal": "^2.10",
        "fideloper/proxy": "^4.0",
        "hashids/hashids": "^4.1",
        "laravel/framework": "^7.0",
        "laravel/tinker": "^2.0",
        "laravel/ui": "^2.0",
        "livewire/livewire": "^2.0",
        "propaganistas/laravel-phone": "^4.2",
        "spatie/laravel-activitylog": "^3.14",
        "spatie/laravel-permission": "^3.11",
        "te7a-houdini/laravel-trix": "^2.0"
    },
    "require-dev": {
        "barryvdh/laravel-debugbar": "^3.2",
        "barryvdh/laravel-ide-helper": "^2.8",
        "facade/ignition": "^2.0",
        "fzaninotto/faker": "^1.4",
        "garygreen/pretty-routes": "^1.0",
        "mockery/mockery": "^1.0",
        "nunomaduro/collision": "^4.1",
        "phpunit/phpunit": "^8.5"
    },
```

I also tried it whether in production mode or development mode, didn't change anything.

Here is my handle() method:

```
public function handle()
    {
        // Setup progress bar
        $lineCount = Contact::count();
        $bar = $this->output->createProgressBar($lineCount);
        $bar->start();

        $contacts = Contact::get();
        foreach ($contacts as $contact) {
            $this->line("About to process ID: " . $contact->id);
            try {
                $contact->value = PhoneNumber::make($contact->value, ['US'])->formatE164();
                $contact->save();
                $bar->advance();
            } catch (\Exception $e) {
                $this->error("Failed to format, ID: " . $contact->id);
                $bar->advance();
                continue;
            }
        }
    }
```

I'll keep tshooting.

**Comment by tintworldsean** [comment]
I have the same issue actually. @guyinpv - did you ever find a resolve?
